MultiplayerMod/Game/Chores/ChoreConsumerEvents.cs

```diff
@@ -1,11 +1,14 @@
 using System;
 using HarmonyLib;
+using MultiplayerMod.Core.Logging;
 using MultiplayerMod.Multiplayer.State;
 
 namespace MultiplayerMod.Game.Chores;
 
 //[HarmonyPatch(typeof(ChoreConsumer), nameof(ChoreConsumer.FindNextChore))]
-public static class ChoreConsumerEvents {
+public class ChoreConsumerEvents {
```

Comment:
Why not static?

Reply:
LoggerFactory.GetLogger requires class to be not static.